portal/server/constants/index.js

```diff
@@ -1,39 +1,39 @@
+const ALL_BANKS_ID = require('./all-banks-id');
 const DASHBOARD = require('./dashboard');
 const DATE = require('./date');
 const FACILITY_HAS_BEEN_ISSUED = require('./facility-has-been-issued');
 const FACILITY_STAGE = require('./facility-stage');
 const FIELD_NAMES = require('./field-names');
+const { FILE_UPLOAD } = require('./file-upload');
+const HTTP_ERROR_CAUSES = require('./http-error-causes');
+const { LANDING_PAGES } = require('./landing-pages');
+const { MONTH_NAMES } = require('./month-names');
+const PORTAL_URL = require('./portalUrl.constant');
+const { PRIMARY_NAV_KEY } = require('./primary-nav-key');
 const PRODUCT = require('./product');
 const STATUS = require('./status');
 const SUBMISSION_TYPE = require('./submission-type');
 const TRANSACTION_STAGE = require('./transaction-stage');
 const TRANSACTION_TYPE = require('./transaction-type');
-const PORTAL_URL = require('./portalUrl.constant');
 const SORT_BY = require('./sort');
```

Comment:
How come the ordering here has changed? Doesnt need reverting just a bit odd

Reply:
I updated them to be ordered alphabetically.